.ci/static-checks.sh

```diff
@@ -1090,6 +1114,23 @@ main()
 			fi
 		fi
 
+		if [ "${RUST_AGENT:-}" == "yes" ]; then
+			# skip checks specific to go
+			if [[ "${func}" =~ "go_arch_specific" ]]; then
```

Comment:
If someone  modify this function names, the rust agent CI will be broken easily

Reply:
fixed